src/services/accommodation.service.js

```diff
@@ -0,0 +1,61 @@
+import db from '../database/models/index';
```

Comment:
default export should be a const named `AccommodationService`

Reply:
I applied the changes